web/js/codeworld.js

```diff
@@ -248,6 +248,23 @@ function help(doc) {
     });
 }
 
+function editorHelp(doc) {
+    var helpText = "<h3>Editor Shortcuts</h3>" +
+        "<div id='keyboard-shortcuts'><table><tbody>" +
+        "<tr><td>Ctrl + Space </td><td> Autocomplete</td></tr>" +
+        "<tr><td>Ctrl + Up </td><td> Zoom In </td></tr>" +
+        "<tr><td>Ctrl + Down </td><td>  Zoom Out </td></tr>" +
+        "</tbody></table></div>";
```

Comment:
Also: Shift-Space is an alternative for auto-complete.  (Added because Chromebooks sometimes steal Ctrl-Space for swapping between US and Intl keyboard layout)